specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json

```diff
@@ -702,6 +750,25 @@
     }
   },
   "definitions": {
+    "QuotaGet": {
+      "description": "Subscription quotas",
```

Comment:
>Subscription quotas [](start = 22, length = 19)

if this is a tenant level API this is no longer accurate.  Does an entire tenant really share quota?

Reply:
you are correct, this was a confusion on my part.  Updated this pr and the manifest to remove the Tenant routing.